.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,73 @@
+name: "\U0001F41E Bug report"
+description: Report an issue with Vite
+labels: [pending triage]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is. If you intend to submit a PR for this issue, tell us in the description. Thanks!
+      placeholder: Bug description
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Describe the bug
+      description: Please provide a link to a repo that can reproduce the problem you ran into. A reproduction is required unless you are absolutely sure that the issue is obvious and the provided information is enough to understand the problem. If a report is vague (e.g. just a generic error message) and has no reproduction, it will receive a "need reproduction" label. If no reproduction is provided after 3 days, it will be auto-closed.
+      placeholder: Reproduction
+    validations:
+      required: true
+  - type: textarea
+    id: system-info
+    attributes:
+      label: System Info
+      description: Output of `npx envinfo --system --npmPackages vite,@vitejs/plugin-vue --binaries --browsers`
+      render: shell
+      placeholder: System, Binaries, Browsers
+    validations:
+      required: true
+  - type: dropdown
+    id: package-manager
+    attributes:
+      label: Used Package Manager
+      description: Select the used package manager
+      options:
+        - npm
+        - yarn
+        - pnpm
```

Comment:
Should we be specific with npm 6/7 and yarn 1/2? It's possible that the user might have two versions of a package manager installed, although probably unlikely.

Reply:
This info is also in the binaries info, no?

Reply:
Seems like although yarn 1 and 2 can coexist, the binaries info does correctly detect which version is in use for the project. npm 7 seems to overwrite 6 entirely, so probably not an issue.